src/main/java/core/basesyntax/suppliers/FigureSupplier.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax.suppliers;
+
+import core.basesyntax.abstractandenum.Color;
+import core.basesyntax.abstractandenum.Figure;
+import core.basesyntax.models.Circle;
+import core.basesyntax.models.IsoscelesTrapezoid;
+import core.basesyntax.models.Rectangle;
+import core.basesyntax.models.RightTriangle;
+import core.basesyntax.models.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final int getRandomInt = new Random().nextInt(100);
+    private final double getRandomDouble = new Random().nextDouble();
+    private final int figureIndex = new Random().nextInt(4);
```

Comment:
```suggestion
    private final Random random = new Random();
```
randomize a new value in methods
add constants for 100 and 4